### PR TITLE
Changelog for 8.0 beta 1

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Update Changelog for 8.0.0 beta 1 along with contributors
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| Related PRs       | ~
| How to test?      | ~
| Possible impacts? | ~

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -46,16 +46,18 @@ GitHub contributors:
...
 - Amoifr
 - Anas Mammeri
 - anat
 - Anatole
 - Anders Bilfeldt
 - Andrew
 - andromaque
-- Amin-Hosseini
-- Amit Kumar Tiwari
+- Andromaque
```
Comment (eternoendless) [inline CONTRIBUTORS.md]: Watch out, this one appears twice

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -129,13 +137,15 @@ GitHub contributors:
...
 - Christian Verardi
+- Christophe Rouille
 - Christophe Zarebski
 - ChristopheBoucaut
 - CINS
 - cippest
 - cirykpopeye
 - Claire Meuriot
 - Clotaire 202 ecommerce
+- Clotaire Renaud
```
Comment (eternoendless) [inline CONTRIBUTORS.md:146]: This one is the same as Clotaire 202 ecommerce

**Comment by kpodemski** [comment]
In the contributors' list,`Sam` should be `Sam Berry`

File: docs/CHANGELOG.txt [hunk 3]
```diff
@@ -21,6780 +21,1022 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 @copyright Since 2007 PrestaShop SA and Contributors
 @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
 
-Release Notes for PrestaShop
+Release Notes for Changelog PrestaShop 8
```
Comment (eternoendless) [inline docs/CHANGELOG.txt]:
```suggestion
Changelog for PrestaShop 8
```

**Comment by eternoendless** [comment]
> In the contributors' list,`Sam` should be `Sam Berry`

How do you know @kpodemski ?

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @jolelievre

**Pull request merged**